pom.xml

```diff
@@ -4,14 +4,27 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.596.1</version>
+        <version>3.50</version>
         <relativePath />
     </parent>
     <artifactId>liquibase-runner</artifactId>
     <version>1.4.0-SNAPSHOT</version>
     <name>Liquibase Runner</name>
 
     <packaging>hpi</packaging>
+    <properties>
+        <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
+        <jenkins.version>2.223</jenkins.version>
```

Comment:
I would recommend to target the LTS release if possible. Better to support a few LTS baselines